### DISCUSSION TITLE
Collapsable relationships doesn't make api calls when collapsed

### DISCUSSION THREAD

**Discussion by oskobri** [laravel/nova-issues#5221] [Ideas & Feature Requests]
It would be cool if collapsed relationships don't make an API call on the first load of the page resource because the API call will be made when the relations are extended anyway.

**Comment by davidhemphill** [comment]
During my testing, I could not see any relation XHR requests being made.

**Reply by oskobri** [reply]
I see that is not the search relationships endpoint but the filters endpoint so it's not that important.
